### DISCUSSION TITLE
Is recording calorie burning from classes/events necessary?

### DISCUSSION THREAD

**Discussion by shashank-madan** [aditikilledar/burnout-proj3#38] [General]
This is in reference to issue #16.
Will it be an accurate representation of calories being burnt?

**Comment by aditikilledar** [comment]
Yess! I think it will be useful for the user to track their calorific spending.

**Reply by shashank-madan** [reply]
How would we implement it?
Is each person participating in the class burning the same amount of calories?
Will we have to record as the date of the class expires and record automatically?

**Reply by aiyer786** [reply]
We can add a button in the upcoming events tab where it says "Marks as complete". We can take into account the average calories burned during that event as the calories burnt for all the users. 
We should not mark as complete automatically because we have to make sure that the user has attended the class for sure.

**Reply by shashank-madan** [reply]
Good Idea. Will comment on the original issue
